launchflow/cli/main.py

```diff
@@ -120,6 +120,12 @@ def _set_global_project_and_environment(
 """
 
 
+# Flow should be
+# Detect all frameworks
+# Prompt user to select their cloud provider
+# Prompt user to select a frameworks to deploy
+# Prompt user to select runtime
+# Prompt user to select build
```

Comment:
did you mean to leave these comments? aside: we should sketch out the ideal cli flow on a doc / maybe stumble it before we start coding it up

Reply:
yea I just left them here for reference from what we talked about